src/index.css

```diff
@@ -3,3 +3,9 @@ body {
   padding: 0;
   font-family: sans-serif;
 }
+table, th, td {
```

Comment:
This style will apply to all the `table`, `th` and `td` elements in the project. This will also affect any other table that may be added in the future. Instead, you should use [a class selector](https://developer.mozilla.org/en-US/docs/Web/CSS/Class_selectors) to differentiate the `CarsDataTable` elements from generic ones.